### PR TITLE
Placeholder ember-cli notes 26 october 2017

### PR DESCRIPTION
**Pull request by hjdivad**

### CODE REVIEW

File: ember-cli/2017-10/oct-19.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# ember-cli Meeting Notes - 2017-10-26
+
+## Attendees
+
+
+## Agenda
```
Comment (hjdivad) [inline ember-cli/2017-10/oct-19.md]: please reply here with agenda items

Comment (hjdivad) [reply ember-cli/2017-10/oct-19.md]: - https://github.com/ember-cli/rfcs/pull/112
- https://github.com/ember-cli/ember-cli/pull/7401

Comment (kellyselden) [reply ember-cli/2017-10/oct-19.md]: https://github.com/ember-cli/ember-cli/pull/7388

Comment (kellyselden) [reply ember-cli/2017-10/oct-19.md]: https://github.com/ember-cli/ember-cli/pull/7384
https://github.com/ember-cli/ember-cli/pull/7382

Comment (kategengler) [reply ember-cli/2017-10/oct-19.md]: Merging RFC process & repo with emberjs/rfcs

**Comment by hjdivad** [comment]
cc @rwjblue @turbo87 @kellyselden @kategengler

**Comment by Turbo87** [comment]
thanks @hjdivad !

**Pull request merged**